### PR TITLE
Merchant items index page

### PR DESCRIPTION
**Pull request by klatour324**
Started working on the merchant items index portion and created the index pages for merchant items.
Updated the naming of our files to be merchant_items instead of items to be more explicit.
Added necessary route for merchant items and view that now contains a list of item names for a merchant.

### CODE REVIEW

File: spec/features/merchants/merchant_items_index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                                    unit_price: 150.00)
+
+
+      visit "/merchant/#{merchant.id}/items"
+
+      within("#merchant-items") do
+        expect(page).to have_content(item1.name)
+        expect(page).to have_content(item2.name)
+        expect(page).to have_content(item3.name)
+        expect(page).to_not have_content(item4.name)
```
Comment (jequeza) [inline spec/features/merchants/merchant_items_index_spec.rb:24]: Loving these edge case tests!

**Comment by jequeza** [comment]
Great work Dev! 👍
